Source/Diagnostics/BoostedFrameDiagnostic.H

```diff
@@ -109,9 +167,10 @@ public:
 
     BoostedFrameDiagnostic(amrex::Real zmin_lab, amrex::Real zmax_lab,
                            amrex::Real v_window_lab, amrex::Real dt_snapshots_lab,
-                           int N_snapshots, amrex::Real gamma_boost,
+                           int N_snapshots, amrex::Real dt_slice_snapshots_lab,
+                           int N_slice_snapshots, amrex::Real gamma_boost,
                            amrex::Real t_boost, amrex::Real dt_boost, int boost_direction,
-                           const amrex::Geometry& geom);
+                           const amrex::Geometry& geom, amrex::RealBox& slice_realbox);
 
     void Flush(const amrex::Geometry& geom);
```

Comment:
Could you add a quick description (1 line is often enough) to all these member functions? For instance, a reader might not understand why there is both a `Flush` and a `write` function, and the reason is that
 - `writeLabFrameData` fills the buffers and writes to disk when buffers are full
 - `Flush` is only called at the end of the simulation, and writes the buffers even if they're not full.

Reply:
Actually `writeLabFrameData` is doing a lot of things, could you briefly describe it (either in the `cpp` of `H` file)? I know this is not really linked to your PR, but it's always good to add helpful comments :)

Reply:
I have added a brief description in the .H file. 
Please let me know if you have any additions/suggestions